shared/notifications/index.js

```diff
@@ -0,0 +1,94 @@
+import { getSwapsFeatureLiveness } from '../../ui/app/ducks/swaps/swaps';
+import { getSwapsEthToken, getIsMainnet } from '../../ui/app/selectors';
+import { BUILD_QUOTE_ROUTE } from '../../ui/app/helpers/constants/routes';
+
+export const UI_NOTIFICATIONS = {
```

Comment:
Are we planning on localizing the copy for the notifications? I suppose for very new announcements it may be tricky to get translations ordered that quickly

Reply:
Good point! These should be localized. It should at least be possible for us to use any translations that we do have.

Reply:
Addressed in 100cab050